[BugFix] build script supports Python3 and fix file path typos (#9563)

packy92 opened the pull request:
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：
<!--
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #9563

## Problem Summary(Required) ：
<!-- (Please describe the changes you have made. In which scenarios will this bug be triggered and what measures have you taken to fix the bug?) -->
1. add python3 support in env.sh.
2. Add a default python execute command property and update it when user set a PYTHON environment variable. Use this property in antrun-plugin to ensure build successfully in platform with Python2.x installed or PYTHON environment variable had been set. This won't help user who only installed Python3 and without setting PYTHON environment variable.

packy92 committed:
[BugFix] build script supports Python3 and fix file path typos (#9563)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/StarRocks/starrocks?pullRequest=9571) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/StarRocks/starrocks?pullRequest=9571) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/StarRocks/starrocks?pullRequest=9571) it.</sub>

imay reviewed (dismissed):
LGTM

kangkaisen commented:
run starrocks_fe_unittest

wanpengfei-git commented:
run starrocks_admit_test

stdpain commented:
in other planform won't load env.sh

wanpengfei-git commented:
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

stdpain commented on fe/fe-core/pom.xml:
```diff
@@ -828,15 +828,15 @@ under the License.
                                     <arg value="${starrocks.home}/gensrc/proto/lake_service.proto"/>
                                 </exec>
-                                <exec executable="python">
+                                <exec executable="${PYTHON}">
```
suggest make env PYTHON as a variable

packy92 replied:
like this `<exe exec executable="${PYTHON}">` ?

stdpain replied:
you can ref thrift.

```
<profile>
  python3 profile:
<profile>
```

packy92 replied:
please review the latest commit.

packy92 committed:
[BugFix] build script supports Python3 and fix file path typos (#9563)

Add a default python execute command property and update it when user set a PYTHON environment variable. Use this property in antrun-plugin to ensure build successfully in platform with Python2.x installed or PYTHON environment variable had been set. This won't help user who only installed Python3 and without setting PYTHON environment variable.

wanpengfei-git commented:
run starrocks_admit_test

wanpengfei-git commented:
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)
